stream-legal-entity/legal-entity-core/src/test/java/com/backbase/stream/LegalEntitySagaTest.java

```diff
@@ -134,8 +135,9 @@ void customServiceAgreementCreation() {
         User adminUser = new User().internalId("someAdminInId").externalId(adminExId);
         legalEntity = new LegalEntity().internalId(leInternalId).externalId(leExternalId).addAdministratorsItem(adminUser)
             .parentExternalId(leParentExternalId).customServiceAgreement(customSa).users(singletonList(regularUser))
-            .productGroups(singletonList(productGroup)).subsidiaries(singletonList(
-                    new LegalEntity().externalId(leExternalId).customServiceAgreement(customSa)
+            .productGroups(singletonList(productGroup)).subsidiaries(List.of(
+                    new LegalEntity().externalId(leExternalId).customServiceAgreement(customSa),
+                    new LegalEntity().externalId(leExternalId2).customServiceAgreement(customSa).createInDBS(false)
```

Comment:
this unit test does nothing, where are you checking it wasn't created?

Reply:
added assertions.